POSIX uses -1 and does not define WAIT_ANY so we need to define it if
needed.

See:
http://pubs.opengroup.org/onlinepubs/9699919799/functions/waitpid.html
http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/sys_wait.h.html

--- openvswitch-2.5.0/tests/test-ovn.c
+++ openvswitch-2.5.0/tests/test-ovn.c.new
@@ -19,6 +19,11 @@
 #include <errno.h>
 #include <getopt.h>
 #include <sys/wait.h>
+
+#ifndef WAIT_ANY
+#define WAIT_ANY (-1)
+#endif
+
 #include "dynamic-string.h"
 #include "fatal-signal.h"
 #include "match.h"
